helpers/image_helper.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+require 'open-uri'
+require_relative '../config/StaticData'
+
+# Class for working with images after convert service
+class ImageHelper
+  def self.get_image_size(url)
+    temp_file = Tempfile.new('images-', StaticData::TMP_FOLDER.to_s, encoding: 'ascii-8bit')
```

Comment:
I'm not sure, but I think encoding is not needed, since it's not some base64 encoding, but plain ascii